docs/tutorials/aws-sd.md

```diff
@@ -226,6 +226,16 @@ spec:
     ...
 ```
 
+## Enable sub-domain hosted zone match
+The aws-zone-match-parent flag extends external-dns to support sub-domain matching using its parent domain.
+
+```yaml
+## hosted zone domain: domain.com
+--domain-filter=x.sub-domain.domain.com,sub-domain.com
```

Comment:
`domain.com` is not parent of `sub-domain.com`
Can you explain more in the doc what this new flag will do differently?

Reply:
This new flag will extend the domain-filter function adding subdomain matching by its parent. For example, now if a hosted zone `example.com` is used in the --domain-filter but as parent `x.example.com`, with the `--aws-zone-match-parent` flag, the `external-dns will` also verify its parent. In case, there is a parent, the records are created in that hosted zone.
`x.example.com` -- it is not a hosted zone, then check the parent
`example.com` -- it is a hosted zone, then create the records.